artifactory/commands/buildinfo/publish.go

```diff
@@ -91,12 +94,51 @@ func (bpc *BuildPublishCommand) Run() error {
 	if err != nil {
 		return err
 	}
+
+	buildTime, err := time.Parse(buildinfo.TimeFormat, buildInfo.Started)
+	if errorutils.CheckError(err) != nil {
+		return err
+	}
+
+	artVersion, err := servicesManager.GetVersion()
+	if err != nil {
+		return err
+	}
+	artVersionToStr, err := strconv.Atoi(artVersion[0:1])
```

Comment:
The name of the ```artVersionToStr``` is wring, because it does not include "artifactory's version as string". It includes the first version digit as an integer and not as a string.
Here's wat I suggest:
1. Let's get the Artifactory's major version (and not the first digit) as an int.
2. Let's name the variable that stores it ```majorVersion```.